app/moves/middleware.js

```diff
@@ -116,24 +115,42 @@ const movesMiddleware = {
           return moveService
             .getMovesCount({ dateRange, status: moveType.filter, locationId })
             .then(count => {
-              const { label } = moveType
               return {
-                label,
+                ...moveType,
                 value: count,
                 active: moveType.filter === req.params.status,
                 href: `${req.baseUrl}/${period}/${date}${
                   locationId ? '/' + locationId : ''
                 }/${moveType.filter}`,
               }
             })
-            .then(presenters.moveTypesToFilterComponent)
         })
       )
+      res.locals.moveTypeNavigation.map(presenters.moveTypesToFilterComponent)
```

Comment:
Any reason why this can't be the same as before?

Reply:
You asked for `.map` to be moved outside the presenter. I am not sure I understand what you mean.

Reply:
Hmm -- that line makes no sense, you're right.